extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -504,7 +504,7 @@ export class GmailElementReplacer implements WebmailElementReplacer {
 
   private filterAttachments = (potentialMatches: JQueryEl | HTMLElement, regExp: RegExp) => {
     return $(potentialMatches).filter('span.aZo:visible, span.a5r:visible').find('span.aV3').filter(function () {
-      const name = this.innerText.trim();
+      const name = this.innerHTML.trim();
```

Comment:
I feel like this could break other attachments with special characters in them, like `a<b` which will get translated to `a&lt;b` ? Or maybe `a&b` etc?

Reply:
Right. Apparently, jQuery's `.text()` returns what we need here:

![image](https://user-images.githubusercontent.com/6059356/118555139-fb920800-b76a-11eb-9bef-220b48bb364d.png)